### PR TITLE
Use $SCRIPT_DIR instead of ./ for generic script locations

### PR DESCRIPTION
**Pull request by kb100**
Fixes https://github.com/vivien/i3blocks-contrib/issues/22

### CODE REVIEW

**Comment by kb100** [comment]
@jpleau @vivien can either of you take a quick look and merge this? I don't want to merge my own pull request.

**Comment by jpleau** [comment]
@kb100 I'll take a look tonight, sorry for the lack of inactivity

If all is ok, all I have to do is rebase it against master, then merge it
in master and add a little "#26" somewhere?

2016-01-04 16:18 GMT-05:00 kb100 <notifications@github.com>:

**Comment by kb100** [comment]
After looking at it, all you have to do is hit the merge pull request button.

**Comment by kb100** [comment]
@jpleau The pull request now makes the convention of using $SCRIPT_DIR explicit and uniform throughout the repo.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,17 @@ contrib | core
...
 master  | master
 
-You may want to take a look at the individual blocklet directories or 
-the [contribution guidelines](https://github.com/vivien/i3blocks-contrib/blob/master/CONTRIBUTING.md).
+You may want to take a look at the individual blocklet directories,
+which contain descriptions, screenshots, and installation/configuration 
+instructions.
+Note: configurations reference `$SCRIPT_DIR`, meaning the directory that you put
+the script into.
+`$SCRIPT_DIR` is not an environmental variable, you must fill it in
```
Comment (jpleau) [inline README.md]: $SCRIPT_DIR *can* be an environment variable, in fact on my system it is. I put my custom scripts in there and use `command=$SCRIPT_DIR/my_script`

**Comment by jpleau** [comment]
Regarding my comment, here's a quick test:

i3blocks.conf:
```
[my_test]
command=$SCRIPT_DIR/test
```

`env SCRIPT_DIR=/home/jason/scripts i3blocks`

And /home/jason/scripts/test gets executed

**Comment by kb100** [comment]
@jpleau You are right, and perhaps that sentence needs rewording. What I am trying to say is that i3blocks does not determine what `$SCRIPT_DIR` should be for each script, it only exists if you set it.

Indeed, I do the same as you, setting `$SCRIPT_DIR` in my i3blocks call and this works in bash as well.

**Comment by jpleau** [comment]
Cool ! When I get home I'll re-test all the other blocklets to make sure they all still work with your changes to their `command=` line. If you could reword the use of SCRIPT_DIR that'd be great

Once all is done I'll merge

Cheers !

**Comment by jpleau** [comment]
@kb100 I'm not sure how we could reword it, we need to explain that it can be set as an environment variable, or that you can directly replace the `$SCRIPT_DIR` part with an hardcoded path. 

Maybe give examples on how to set the env. variable, and an example where the path is replaced directly? 

Not sure if that's that useful though..

**Comment by kb100** [comment]
@jpleau See the latest version. Instead of crowding the main README I decided to explain `$SCRIPT_DIR` on the FAQ and just link to it in the main README.

**Comment by jpleau** [comment]
LGTM, merged!

Cheers

**Pull request merged**